app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -1,11 +1,21 @@
 /* list
 ---------------------------------------------------------------------------- */
+$badge-width: 24px;
```

Comment:
Did you intend to define this twice?  Also if it is actually used my multiple modules it may be a good candidate for going into variables.